docs/src/julia_syntax.md

```diff
@@ -160,28 +209,58 @@ BUGSModel (parameters are in transformed (unconstrained) space, with dimension 4
     alpha1
     alpha0
 
-  Variable sizes and types:
-    b: size = (21,), type = Vector{Float64}
-    p: size = (21,), type = Vector{Float64}
-    n: size = (21,), type = Vector{Int64}
-    alpha2: type = Float64
-    sigma: type = Float64
-    alpha12: type = Float64
-    alpha0: type = Float64
-    N: type = Int64
-    tau: type = Float64
-    alpha1: type = Float64
-    r: size = (21,), type = Vector{Float64}
-    x1: size = (21,), type = Vector{Int64}
-    x2: size = (21,), type = Vector{Int64}
-
-julia> SeedsParams(m)
-SeedsParams:
-  r       = [0.0, 0.0, 0.0, 0.0, 39.0, 0.0, 0.0, 72.0, 0.0, 0.0  …  0.0, 0.0, 0.0, 0.0, 4.0, 12.0, 0.0, 0.0, 0.0, 0.0]
-  b       = [-Inf, -Inf, -Inf, -Inf, Inf, -Inf, -Inf, Inf, -Inf, -Inf  …  -Inf, -Inf, -Inf, -Inf, Inf, Inf, -Inf, -Inf, -Inf, -Inf]
-  alpha0  = -1423.52
-  alpha1  = 1981.99
-  alpha2  = -545.664
-  alpha12 = 1338.25
-  tau     = 0.0
+julia> # Create model with observations for r
+julia> r_data = [10, 23, 23, 26, 17, 5, 53, 55, 32, 46, 10, 8, 10, 8, 23, 0, 3, 22, 15, 32, 3]
+julia> m_obs = seeds((r=r_data,), x1, x2, N, n)
+
+julia> # Using unflatten for initialization with missing values
+julia> using JuliaBUGS: unflatten
+julia> params = unflatten(of(SeedsParams), missing)
```

Comment:
Why do I still need `of(SeedsParams)` when `SeedsParams` was created with `@of`?